kolibri/content/apps.py

```diff
@@ -0,0 +1,9 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+from django.apps import AppConfig
+
+
+class KolibriContentConfig(AppConfig):
+    name = 'kolibri.content'
+    label = 'kolibricontent'
+    verbose_name = 'Kolibri Content'
```

Comment:
This wasn't essential to the other stuff in this PR, but it provides a base skeleton for app config (paralleling the one in `auth`).